test/create_db.sh

```diff
@@ -22,33 +22,28 @@ mysql $dbconn -e "SET GLOBAL max_connections = 500;"
 for dbenv in $DBENVS; do
   db="boulder_sa_${dbenv}"
 
-  if mysql $dbconn -e 'show databases;' | grep $db > /dev/null; then
-    echo "Database $db already exists - skipping create"
-  else
-    create_script="drop database if exists \`${db}\`; create database if not exists \`${db}\`;"
-
-    mysql $dbconn -e "$create_script" || die "unable to create ${db}"
+  create_script="drop database if exists \`${db}\`; create database if not exists \`${db}\`;"
 
-    echo "created empty ${db} database"
-  fi
+  mysql $dbconn -e "$create_script" || die "unable to create ${db}"
 
-  goose -path=./sa/_db/ -env=$dbenv up || die "unable to migrate ${db} with ./sa/_db/"
-  echo "migrated ${db} database with ./sa/_db/"
+  echo "created empty ${db} database"
 
-  if [[ "$BOULDER_CONFIG_DIR" = "test/config-next" ]]; then
-    nextDir="./sa/_db-next/"
+  if [[ "$BOULDER_CONFIG_DIR" = "test/config" ]]; then
```

Comment:
Maybe better to do this the other way around, so in the case that BOULDER_CONFIG_DIR isn't set at all, we end up testing what's actually in production? Or maybe this was is the best, so we default to testing the wildest possible set of things? Honestly not sure, interested in others' thoughts.

Reply:
yeah, I went this direction on the latest push.